server/views/admin-add-category.hbs

```diff
@@ -10,7 +10,7 @@
   <div class="form-group row has-success">
     <label for="example-text-input text-success " class="col-12 col-form-label"><h3 class="text-primary">Icon:</h3></label>
     <div class="col-lg-6 col-md-8 col-sm-9 col-xs-9 ">
-      <input class="form-control" type="text" name="icon" id="example-text-input">
+      <input class="form-control" type="text" name="icon" id="example-text-input" value="default-icon">
```

Comment:
no, that's not good - we should not prefill it. use `placeholder='default-icon'` if you'd like to give suggestion, and show an error if they don't enter a value